xc/common/utils/prjxray_create_ioplace.py

```diff
@@ -9,6 +9,13 @@
 from lib.parse_pcf import parse_simple_pcf
 
 
+def get_synth_tile_from_pad(synth_tiles, pad):
+    for _, tile in synth_tiles['tiles'].items():
+        for pin in tile['pins']:
+            if pin['pad'] == pad:
+                return tile
```

Comment:
Should have an error if this falls off the end? Also docstring?

Reply:
I don't think this should have an error based on the discussion below.  Is that correct?

Reply:
Agree in principle.  I believe it is worth having an explicit comment that it is okay to return `None`, because ...